content/departments/engineering/dev/admin-exp/devops/managed/v1.1/mi1-1_enable_executors_process.md

```diff
@@ -43,38 +43,12 @@ mg executors set-token --token $(terraform output -raw executor_proxy_password)
 
 ### Confirm executors is actually working
 
-Increase the minimal replica of executors auto scaling group to `1`
-
-```diff
--executor_min_replicas              = 0
-+executor_min_replicas              = 1
-```
-
-Apply the change
-
-```sh
-terraform apply
-```
-
-Then run the check, which will retry until either the executors are up or a 5 minute timeout has been reached, after which you should check configuration errors:
+Then run the check, which will increase the minimal replica of executors auto scaling group to `1` and retry until either the executors are up or a 5 minute timeout has been reached, after which you should check configuration errors:
```

Comment:
This is a little confusing since it says "run the check" but doesn't specify what that means. Presumably it means run `mg executors check` but the way the sentence is written
> after which you should check configuration errors

makes it seem like `mg executors check` should only be run afterwards.

Reply:
I think more clearly describing exactly what to run here would make sense.

e.g. "Run `mg executors check` which will increase the count ... After it's complete, rerun `mg executors check` to check for configuration errors"